Beautify README file. (#6)
Tolga1452/Vencord: README.md

`😺` A stupid cat programmer's crazy Vencord plugins.

`🗨️` Join our [Discord Server](https://discord.gg/PxDj9XeHkB) for more information.

# Frequently Asked Questions

## Is this the original Vencord client modification?
No, this is a fork of Vencord with unofficial plugins. You can find the original repository of Vencord [here](https://github.com/Vendicated/Vencord).

> ---

Reviewer: Separators not needed here
```suggestion
```

## Is this fork safe?
Yes, this repository is safe. If you don't trust it, feel free to look at the source code.

> ---

Reviewer: ```suggestion
```

## Is this fork up-to-date with original Vencord?
Yes, it usually is, but we don't update this fork immediatley after the original Vencord is updated. You'll have to update it manually. We release a new version when there are necessary changes.

---

> # Dependencies

Reviewer: Why did you try to confuse people even more?
```suggestion
# Setup

## Requirements
```

> 1. Download the latest version of Node.js from [here](https://nodejs.org/en/download/current).

Reviewer: If they are not steps, this could be better
```suggestion
 - Download the latest version of Node.js from [here](https://nodejs.org/en/download/current).
```

> 2. Download Git from [here](https://git-scm.com/download).
> 3. Open a terminal and run the following commands:

Reviewer: ```suggestion
- Download Git from [here](https://git-scm.com/download).
- Open a terminal and run the following commands:
```

> # Installing

Reviewer: ```suggestion
## Installing
```

You can watch [this](https://drive.google.com/file/d/1zlWv4t14ORqy7QuF_Sup7DLkf7pAgS90/view?usp=sharing) tutorial to make it more easy.

> ---

Reviewer: I know why you wanted to separate it but this also ends the context too
```suggestion
```

> Install `pnpm`:

Reviewer: Why no headers here :skull:
```suggestion
### Install `pnpm`:
```

> [!IMPORTANT]
> This next command may need to be run as admin/root depending on your system, and you may need to close and reopen your terminal for `pnpm` to be in your PATH.

```shell
npm i -g pnpm
```

> ---

Reviewer: ```suggestion
```

> [!IMPORTANT]
> Make sure you aren't using an admin/root terminal from here onwards. It **will** mess up your Discord/Vencord instance and you **will** most likely have to reinstall.

> Clone Vencord:

Reviewer: ```suggestion
### Clone the fork
```

```shell
git clone https://github.com/Vendicated/Vencord
cd Vencord
```

> ---

Reviewer: ```suggestion
```

> Install the dependencies needed:

Reviewer: ```suggestion
### Install the dependencies needed
```

```shell
pnpm install --frozen-lockfile
```

> ---

Reviewer: ```suggestion
```

> Build this fork of Vencord:

Reviewer: ```suggestion
### Build Vencord
```

```shell
pnpm build
```

> ---

Reviewer: ```suggestion
```

> Inject Vencord into your client:

Reviewer: ```suggestion
### Inject Vencord into your Discord client
```

> [!TIP]

> > If you want no errors about permissions, you can close your Discord client by exiting it via the tray. But, if you don't want to, that's fine. Either way works.

Reviewer: ```suggestion
> If you having any errors while injecting, close the running Discord client and try again.
```

```shell
pnpm inject
```

> ---

Reviewer: You broke markdown rules :skull:
```suggestion

---

```

When you want to update your Discord client to the latest version of this fork, just run `npm run update`.

# Plugins

> ## Modified Plugins

Reviewer: Nothing needed to change here
```suggestion
## Modified
```

### [ClientTheme](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/clientTheme>)

> - Added support for dynamic colour.

Reviewer: Again. Also no british :bleh:
```suggestion
- Added dynamic color support.
```

### [Experiments](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/experiments>)

- Added a toggle to switch the release channel to staging.

> ## Added PLugins

Reviewer: ```suggestion
## Added
```

> - **[Become Catgirl](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/becomeCatgirl>):** Turns your messages into catgirl style.

Reviewer: If you want to edit plugin descriptions, edit the actual ones instead of readme first
```suggestion
- **[Become Catgirl](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/becomeCatgirl>):** Turns your messages into a catgirl message.
```
